docs/web/docs/1-getting_started/2-install_and_configure.mdx

```diff
@@ -5,69 +5,94 @@ toc_max_heading_level: 5
 ---
 
 import AsciinemaWidget from '../../src/components/AsciinemaWidget';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 
 ### Prerequisites
 
-Before starting the setup wizard, you will need the following information:
+- AWS account ID(s)
+- AWS organization ID (if applicable)
+- AWS IAM credentials with admin privileges
+- Knowledge on [AWS Hub and Spoke Roles](/reference/aws_hub_and_spoke_roles)
 
-- Your AWS account ID(s)
-- The ID of the AWS organization you wish to manage (if applicable)
-- AWS IAM credentials with administrative privileges on the AWS account(s) you wish to manage
-- An understanding of [AWS Hub and Spoke Roles](/reference/aws_hub_and_spoke_roles)
+If using AWS Organizations, [enable CloudFormation StackSets trusted access](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-orgs-enable-trusted-access.html).
```

Comment:
We now try to turn it on for them, so it's not necessary to do it themselves (provided they are using admin role).